### ISSUE TITLE
Default Image Based On Gender

### ISSUE THREAD

**Issue by poeticjustice** [SocialEngine/phpv4-feature-requests#111]
I know in past i mentioned about default image option to be similar to PHPfox and its a random colour and takes the initials to form a profile image.

But I also thought it be great to have default images based on gender.

Times are moving, and with social media the right for equality can be big.

Would be great to set a default image based on male or female

**Comment by DonnaScriptTechs** [comment]
Yes I like that idea to be able to show users based on gender but there is a plugin for this that I can't seem to find atm.

**Comment by DonnaScriptTechs** [comment]
Ok it's by ipragmatech https://www.socialengine.com/experts/profile/ipragmatech-solutions-pvt-ltd . It's not "live" yet but if you need this now, you can contact them as they have one.

**Comment by gsf00001** [comment]
Although I see the benefit for some sites, this is another of those settings that should be Site-ADMIN configurable (or do what some Devs do for defaults - they provide a default file then Site-ADMIN may simply copy over it with their own file which I prefer to do since I can ren the original in case I ever need it).  Anyway, some sites (such as the one I'm working on) has absolutely nothing to do with male/female - you're simply a person on the site and no searches or other mechanisms will be displayed for sex code.
